models/Restaurant.ts

```diff
@@ -0,0 +1,39 @@
+import mongoose from "mongoose";
+import Hour from "./Hour";
+import Post from "./Post";
+import FeaturedItem from "./FeaturedItem";
+
+/**
+ * @interface
+ * @typedef {Restaurant} Restaurant Represents a restaurant object interface
+ * @property {string} name Restaurant name
+ * @property {string} bannerPicture Banner picture
+ * @property {string} profilePicture Avatar picture
+ * @property {string} handle Restaurant account handle
+ * @property {string} bio Restaurant bio
+ * @property {string} cuisine Restaurant cuisine type
+ * @property {string} price Restaurant price range
+ * @property {string} address Restaurant address
+ * @property {string} hours Restaurant open hours
+ * @property {Hour} hours Restaurant open hours
+ * @property {string} phone Restaurant phone number
+ * @property {string} website Restaurant website
+ * @property {Post} posts Restaurant updates announcements
+ * @property {FeaturedItem} featured Restaurant featured items
+ */
+export default interface Restaurant {
+    _id?: mongoose.Schema.Types.ObjectId,
+    name: string,
+    bannerPicture?: string,
+    profilePicture?: string,
+    handle: string,
+    bio?: string,
+    cuisine: string,
+    price: string,
+    address: string,
+    // hours?: Hour[],
```

Comment:
Is there a reason these are commented out?

Reply:
I created them originally and realized that we need the individual objects for Hour, Post, and FeaturedItem. I still kept them there trying to remind myself that these three objects belong to Restaurant. I will delete them.

Reply:
Okay, no worries if you want to keep them for now! Just wondering why they were there haha